packages/@coorpacademy-components/src/template/back-office/brand-update/index.js

```diff
@@ -50,90 +49,117 @@ const BrandUpdate = props => {
     iconType: key || 'arrow'
   }));
 
-  const getStyle = isSelected => (isSelected ? style.selectedElement : style.unselectedElement);
-
-  const subTabsView = (_subTabs = []) =>
-    map.convert({cap: false})((subTab, _index) => (
-      <div key={subTab.title}>
-        {subTab.type === 'iconLink' ? (
-          <IconLinkItem
-            {...subTab}
-            styles={getStyle(subTab.selected)}
-            uppercase={false}
-            target={'_blank'}
-          />
-        ) : (
-          <LinkItem {...subTab} styles={getStyle(subTab.selected)} uppercase={false} />
-        )}
-      </div>
-    ))(_subTabs);
-
   const formattedTabsViews = map(tab => (
     <div key={tab.title} className={style.tabs}>
       {subTabsView(tab.tabs)}
     </div>
   ))(items);
 
+  return (
+    <div className={style.leftMenu}>
+      <div className={style.logo}>
+        <a href="/">
+          <img src={logo} />
+        </a>
+      </div>
+      <Accordion tabProps={formattedTabs} theme={'setup'} onClick={onItemClick}>
+        {formattedTabsViews}
+      </Accordion>
+    </div>
+  );
+};
+
+const buildNotifications = notifications => {
+  if (isEmpty(notifications)) {
+    return null;
+  }
+
   const notificationsList = notifications.map((notification, index) => {
     return (
       <div className={style.notification} key={index}>
-        <Notification {...notification} />
+        <Banner {...notification} />
       </div>
     );
   });
+  return <div className={style.notifications}>{notificationsList}</div>;
+};
 
-  const contentView = cont => {
-    if (!cont) return <Loader />;
-    const {type} = cont;
-    switch (type) {
-      case 'form':
-        return <BrandForm {...cont} />;
-      case 'list':
-        return <BrandTable {...cont} />;
-      case 'upload':
-        return <BrandUpload {...cont} />;
-      case 'analytics-dashboards':
-        return <BrandAnalytics {...cont} />;
-      case 'home':
-        return <BrandDashboard {...cont} />;
-    }
-  };
-
-  const detailsView = cont => {
-    if (!cont) return;
-    return <BrandTable {...cont} />;
-  };
+const buildHeader = (header, notifications) => {
+  const headerStyle = isEmpty(notifications) ? style.header : style.headerForNotifications;
+  return (
+    <div className={headerStyle}>
+      <Header {...header} />
+    </div>
+  );
+};
 
+const buildTabs = items => {
+  const selectedTab = pipe(
+    filter(e => e.selected),
+    head,
+    get('tabs'),
+    filter(e => e.selected),
+    head
+  )(items);
```

Comment:
filter => head c'est un find non ? 


```suggestion
 const selectedTab = pipe(
    find(e => e.selected),
    get('tabs'),
    find(e => e.selected)
  )(items);
```

Reply:
```suggestion
    const selectedTab = pipe(find('selected'), get('tabs'), find('selected'))(items);

```